core/shared/src/main/scala/fs2/Pull.scala

```diff
@@ -577,15 +582,15 @@ object Pull extends PullLowPriority {
       haltOnSignal: F[Either[Throwable, Unit]]
   ): Pull[F, O, Unit] = InterruptWhen(haltOnSignal)
 
+  private type Cons[+F[_], +O] = (Chunk[O], Pull[F, O, Unit])
```

Comment:
Can we remove this? I find type aliases very confusing in general. They introduce names without introducing any type safety. I find that to be a bad tradeoff.

Reply:
I use type aliae, mostly to have type signatures shorter and avoid deep nesting.
Just as one would use a local `val` to capture an expression without having to write a lot of nested parens.
Compilers may be good at LISP, but humans need line breaks and short sentences. At least I do :)